develop/api_metadata.rst

```diff
@@ -366,3 +333,203 @@ Caveats:
+   client_hostname        | 
+   client_port            | 51773
+   backend_start          | 2018-10-05 13:27:20.75839+03
+   xact_start             | 2018-10-05 13:27:20.84112+03
+   query_start            | 2018-10-05 13:27:20.867446+03
+   state_change           | 2018-10-05 13:27:20.869889+03
+   wait_event_type        | Client
+   wait_event             | ClientRead
+   state                  | idle in transaction
+   backend_xid            | 
+   backend_xmin           | 
+   query                  | COPY (SELECT count(*) AS count FROM users_table_102038 users_table WHERE true) TO STDOUT
+   backend_type           | client backend
+
+The ``query`` field shows data being copied from a shard into a temporary table to be counted.
+
+.. note::
+
+  If a router query (e.g. single-tenant in a multi-tenant application, ``SELECT * FROM table WHERE tenant_id = X``) is executed without a transaction block, then master_query_host_name and master_query_host_port columns will be NULL in citus_worker_stat_activity.
+
+To see how ``citus_lock_waits`` works, we can generate a locking situation manually. First we'll set up a test table from the coordinator:
+
+.. code-block:: postgres
+
+   CREATE TABLE numbers AS
+     SELECT i, 0 AS j FROM generate_series(1,10) AS i;
+   SELECT create_distributed_table('numbers', 'i');
+
+Then, using two sessions on the coordinator, we can run this sequence of statements:
+
+.. code-block:: postgres
+
+   -- session 1                           -- session 2
+   -------------------------------------  -------------------------------------
+   BEGIN;
+   UPDATE numbers SET j = 2 WHERE i = 1;
+                                          BEGIN;
+                                          UPDATE numbers SET j = 3 WHERE i = 1;
+                                          -- (this blocks)
+
+The ``citus_lock_waits`` view shows the situation.
+
+.. code-block:: postgres
+
+   SELECT * FROM citus_lock_waits;
+
+   -[ RECORD 1 ]-------------------------+----------------------------------------
+   waiting_pid                           | 88624
+   blocking_pid                          | 88615
+   blocked_statement                     | UPDATE numbers SET j = 3 WHERE i = 1;
+   current_statement_in_blocking_process | UPDATE numbers SET j = 2 WHERE i = 1;
+   waiting_node_id                       | 0
+   blocking_node_id                      | 0
+   waiting_node_name                     | coordinator_host
+   blocking_node_name                    | coordinator_host
+   waiting_node_port                     | 5432
+   blocking_node_port                    | 5432
+
+In this example the queries originated on the coordinator, but the view can also list locks between queries originating on workers (executed with Citus MX for instance).
+
+Tables on all Nodes
+-------------------
+
+Citus has other informational tables and views which are accessible on all nodes, not just the coordinator.
+
+.. _pg_dist_authinfo:
+
+Connection Credentials Table
```

Comment:
We also have introduced pg_dist_pol_info. Could you capture that as well? Thanks!